xrpl/core/binarycodec/definitions/definitions.json

```diff
@@ -1,2775 +1,2933 @@
 {
```

Comment:
Please keep the formatting the same in this JSON.

Reply:
Fixed in [509ecd1](https://github.com/XRPLF/xrpl-py/pull/759/commits/509ecd1ddf71abaf99239b508517bfab08bf2eb1)